src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicPackage.vb

```diff
@@ -63,13 +63,13 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
             _comAggregate = Implementation.Interop.ComAggregate.CreateAggregatedObject(Me)
         End Sub
 
-        Protected Overrides Sub RegisterInitializationWork(packageRegistrationTasks As PackageRegistrationTasks)
+        Protected Overrides Sub RegisterInitializeAsyncWork(packageInitializationTasks As PackageLoadTasks)
 
-            MyBase.RegisterInitializationWork(packageRegistrationTasks)
+            MyBase.RegisterInitializeAsyncWork(packageInitializationTasks)
 
-            packageRegistrationTasks.AddTask(
+            packageInitializationTasks.AddTask(
                 isMainThreadTask:=False,
-                task:=Function(progress, packageRegistrationTasks2, cancellationToken) As Task
+                task:=Function(packageInitializationTasks2, cancellationToken) As Task
```

Comment:
Given the local you're calling this on is always in scope, do you really ever need this function param? We're not using static lambdas....

Reply:
I'm not a VB expert, I just type until something compiles for it. What's the suggested change?

Reply:
Just delete the lambda parameter entirely in the function signature; it's already available as a local at all call sites so you're just making more typing generally.